src/helpers/factory/article.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
@rwajon please implement `chance` to avoid hard-coding attribute details. things like the ID, title, description details should be randomly generated by `chance`